modules/utxo-lib/test/bitgo/signature.ts

```diff
@@ -186,21 +189,22 @@ function runTestParseScript(
   });
 }
 
-function assertVerifySignatureEquals(
-  tx: UtxoTransaction,
-  prevOutputs: TxOutput[],
+function assertVerifySignatureEquals<TNumber extends number | bigint>(
+  tx: UtxoTransaction<TNumber>,
+  prevOutputs: TxOutput<TNumber>[],
   value: boolean,
   verificationSettings?: {
     publicKey?: Buffer;
     signatureIndex?: number;
-  }
+  },
+  testOutputAmount = defaultTestOutputAmount as TNumber
```

Comment:
this does not look right - should probably be an argument instead

Reply:
removing default value